src/flash/nor/km1m7xx.c

```diff
@@ -0,0 +1,588 @@
+/***************************************************************************
+ * (C) Nuvoton Technology Corporation Japan 2021                           *
```

Comment:
Please follow the OpenOCD conventional rule:
Copyright (C) 2021 by Nuvoton Technology Corporation Japan

Reply:
Fixed copyright writing just like any other.